http/share.go

```diff
@@ -23,6 +24,30 @@ func withPermShare(fn handleFunc) handleFunc {
 	})
 }
 
+var shareListHandler = withPermShare(func(w http.ResponseWriter, r *http.Request, d *data) (int, error) {
+	id := d.user.ID
+	if d.user.Perm.Admin {
+		id = 0
+	}
+	s, err := d.store.Share.List(id)
```

Comment:
Instead of defining magic constants let's add `All()` method to the store for admin users